index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>project2</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Playfair+Display&family=Playfair+Display+SC&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+<body>
+  <!-- trying to make it so that the picture is to the right and text is laying over it-- referred to "clearfix" and "text on picture" articles on w3schools.com -->
+  <div class="card">
+    <img
+      src="https://cdn.glitch.com/7af50fa5-5455-4b82-8564-57e5e1ea17ac%2FThe%20Boys%20in%20the%20Band.jpg?v=1602748106231"
+      width="512"
+      height="308"
+    />
+
+    <div class="text">
+        <h2>The Boys in the Band</h2>
+
+        <nav>
+          <a href="#"> 2020</a>
+          <span class="divider">|</span>
+          <a href="#"> R</a>
+          <span class="divider">|</span>
+          <span class="time">2h2m</span>
+          <span class="divider">|</span>
+          <a href="#"> LGBTQ+ Movies,</a>
+          <a href="#"> Dramas</a>
+        </nav>
+
+        <p>
+          At a birthday party in 1968 New York, a surprise guest and a drunken
+          game leave seven gay friends reckoning with unspoken feelings and
+          buried truths.
+        </p>
+
+        <h3><a class="button" href="#"> Watch Now</a></h3>
+    </div>
+  </div>
+</body>
+</html>
```

Comment:
I've suggested a couple of improvements here, but want to note that overall your HTML is tidy and well thought out!